### DISCUSSION TITLE
Do I need a separate self-hosted backend for each website I make, or can one hosted backend serve multiple TinaCMS instances/projects?

### DISCUSSION THREAD

**Discussion by lpalazzi** [tinacms/tinacms#4513] [Help]
Currently looking into TinaCMS and would likely opt for the self-hosted option. If I want to build multiple websites with TinaCMS for separate clients, users, etc, would I need to spin up a new self-hosted backend for each project? Or can I use the same backend to serve multiple frontends?

**Comment by 99ansh** [comment]
https://discord.com/channels/835168149439643678/1258038651712049223
